source/assets/stylesheets/components/_section.scss

```diff
@@ -1,35 +1,36 @@
 section {
-	padding: 100px 0;
+  padding: 100px 0;
   @include skew_bar();
-	.section-heading {
-		font-size: 40px;
-		margin-top: 0;
-		margin-bottom: 75px;
-	}
+  h2, .section-heading {
+    font-size: 40px;
+    margin-top: 0;
+    margin-bottom: 75px;
+    text-align: center;
+  }
 }
 
 .section-subheading {
-	font-size: 16px;
-	text-transform: none;
-	font-style: italic;
-	font-weight: 400;
-	margin-bottom: 75px;
+  font-size: 16px;
+  text-transform: none;
+  font-style: italic;
+  font-weight: 400;
+  margin-bottom: 75px;
 }
 
 @media(min-width:768px) {
-	section {
-		padding: 150px 0;
-	}
+  section {
+    padding: 150px 0;
+  }
 }
 
 // Services Section
 
 .service-heading {
-	margin: 15px 0;
-	text-transform: none;
+  margin: 15px 0;
+  text-transform: none;
 }
 
-.section--alternate {
+section.alternate, .section--alternate {
```

Comment:
Indeed, I like semantic. And I think BEM reinvents the wheel that is already built into CSS for no added benefit, and worse code style.

I don't want to write code that goes against my technical beliefs. Even more so for a non-profit project relying on good will. Plus, it's a community-driven project, not a one-man creation. In that context, every participant must be able to voice their opinion, in speech as in code.

That's why I didn't forbid BEM when it was introduced, even though I hate it. And even though it changed the initial standard.

Please be kind, rewind, and accept that people's opinions can peacefully co-exist without ruining the project.